app/models/remote_execution_feature.rb

```diff
@@ -10,6 +10,7 @@ class RemoteExecutionFeature < ApplicationRecord
   friendly_id :label
 
   scope :with_host_action_button, -> { where(:host_action_button => true) }
+  scoped_search :on => :host_action_button
```

Comment:
As of @adamruzicka's answer, I'd keep this out of this PR, as it feels out of scope, especially when not used :)

Reply:
removed :)